src/main/java/pl/put/poznan/madness/rest/models/SortingInput.java

```diff
@@ -1,11 +1,61 @@
 package pl.put.poznan.madness.rest.models;
 
 import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 import pl.put.poznan.madness.logic.SortingAlgorithm;
+import pl.put.poznan.madness.rest.exceptions.InvalidSortInputException;
+import pl.put.poznan.madness.rest.exceptions.MissingPropertyKeyException;
+import pl.put.poznan.madness.rest.exceptions.NotAllCustomObjectHaveSortablePropertyException;
 
-public class SortingInput<T> {
-  public List<T> data;
-  public SortingAlgorithm algorithm;
-  public String property;
+public class SortingInput {
+  public List<Object> data;
+  public SortingAlgorithm algorithm = SortingAlgorithm.Bubble;
```

Comment:
![image](https://user-images.githubusercontent.com/35179235/206509115-62ed5409-4d86-47d1-84dd-dbcee8a49df3.png)
@mudziok jednak miałeś rację, trochę benchmark za każdym razem, więc to trzeba zmienić na tablicę enumów

Reply:
Poprawione w ea1e956